jobs/ci_open_jenkins/build/cgt.groovy

```diff
@@ -17,3 +17,8 @@ new SbtMicroserviceJobBuilder('capital-gains-calculator').
 
 new BuildMonitorViewBuilder('CGT-MONITOR')
```

Comment:
See James' PR, he's added the service to the existing CGT-MONITOR; can you do the same please? Thanks.